tests/telemetry.rs

```diff
@@ -510,7 +510,7 @@ mod outbound_dst_labels {
     // Ignore this test on CI, as it may fail due to the reduced concurrency
     // on CI containers causing the proxy to see both label updates from
     // the mock controller before the first request has finished.
-    // See https://github.com/runconduit/conduit/issues/751
+    // See https://github.com/linkerd/linkerd2/issues/751
```

Comment:
Continuing to use links is OK but maybe we should just shorten these to "issue #X" instead.

Reply:
because the repos are disparate, i think i'll shorten it to the github syntax linkerd/linkerd2#751?

Reply:
> because the repos are disparate, i think i'll shorten it to the github syntax linkerd/linkerd2#751?

I think the links would be better than that. The advantage of what I proposed is to remove the branding from the references. If you don't think that's important then I'd just leave them as links.